charts/rollups-node/templates/deployment.yaml

```diff
@@ -33,6 +33,7 @@ spec:
       initContainers:
         {{- if .Values.validator.initContainers }}
         {{- include "tplvalues.render" ( dict "value" .Values.validator.initContainers "context" $ ) | nindent 8 }}
+          image: {{ include "application.image" . }}
```

Comment:
How is this gonna work?

`initContainers` is a list, we don't know how many `initContainer` the user is gonna define.

For ex. we may have an `initContainer` for the snapshot and to create the database.

Reply:
We shouldn't force a naming, the user could provide a snapshot any other way without even using an `initContainer`, for ex.

The approach in this https://github.com/cartesi/helm-charts/pull/83#discussion_r1482480224 is the easy path for now.